### PR TITLE
Add more translations and fixes, remove outdated comments

### PR DESCRIPTION
**Pull request by preco21**
* Translates content-tracing-ko.md
* Remove outdated comments about remote buffer in browser-window.md
* Fix typos and improve grammer in tutorials at KO translations

### CODE REVIEW

**Comment by zcbenz** [comment]
Since we have move the translated docs into a new folder, can you rename the files from `xx-ko.md` to `xx.md`? So you don't have to also modify the links in the docs in future.

**Comment by preco21** [comment]
I see that.
I'm going to work this now :+1:

**Comment by preco21** [comment]
@zcbenz Finished!

File: vendor/node [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 205b013ac86e5500678a791cd54f305580fa4f4b
+Subproject commit 902090af5375e497dded310575f19de5328a9bbc
```
Comment (zcbenz) [inline vendor/node]: Seems that you have included an unrelated change.

Comment (preco21) [inline vendor/node]: Oh it's looks like my mistake.. how can I resolve this?

**Comment by zcbenz** [comment]
Thanks!
